.changeset/thin-rabbits-wait.md

```diff
@@ -0,0 +1,18 @@
+---
+"@astrojs/vue": minor
+---
+
+Adds a `devtools` option
+
+Enabling `devtools` will enable the [official Vue Devtools](https://github.com/vuejs/devtools-next) in development:
```

Comment:
```suggestion
You can enable the [official Vue DevTools](https://devtools-next.vuejs.org/) while working in development mode by setting `devtools:true` in your `vue()` integration config:
```
Even their GitHub link seems to just point to the fancier page, so I'd suggest just linking there directly!